src/components/notebookRenderStrategy.tsx

```diff
@@ -42,6 +46,23 @@ export class NotebookRenderStrategy implements ExpandableNodeRenderStrategy {
 	}
 
 	getChildren(childrenLevel: number): JSX.Element[] {
+		if (typeof (this.notebook.apiHttpErrorCode) === 'number') {
+			const errorString = Strings.getError(this.notebook.apiHttpErrorCode);
```

Comment:
Instead of using this method, why not get the error message from the API?

Reply:
Fixed for this and sharedNotebooks